rasa/core/policies/embedding_policy.py

```diff
@@ -1,694 +1,139 @@
-import copy
-import json
 import logging
-import os
-import pickle
-
-import numpy as np
-from typing import Any, List, Optional, Text, Dict, Tuple
-
-import rasa.utils.io
-from rasa.core.domain import Domain
-from rasa.core.featurizers import (
-    TrackerFeaturizer,
-    FullDialogueTrackerFeaturizer,
-    LabelTokenizerSingleStateFeaturizer,
-    MaxHistoryTrackerFeaturizer,
+from typing import Any, Dict, Optional, Text
+
+from rasa.core.constants import DEFAULT_POLICY_PRIORITY, DIALOGUE
+from rasa.core.featurizers import TrackerFeaturizer
+from rasa.core.policies.ted_policy import TEDPolicy
+from rasa.constants import DOCS_URL_POLICIES
+from rasa.utils.tensorflow.constants import (
+    LABEL,
+    HIDDEN_LAYERS_SIZES,
+    TRANSFORMER_SIZE,
+    NUM_TRANSFORMER_LAYERS,
+    NUM_HEADS,
+    MAX_SEQ_LENGTH,
+    BATCH_SIZES,
+    BATCH_STRATEGY,
+    EPOCHS,
+    RANDOM_SEED,
+    RANKING_LENGTH,
+    LOSS_TYPE,
+    SIMILARITY_TYPE,
+    NUM_NEG,
+    EVAL_NUM_EXAMPLES,
+    EVAL_NUM_EPOCHS,
+    NEG_MARGIN_SCALE,
+    REGULARIZATION_CONSTANT,
+    SCALE_LOSS,
+    USE_MAX_NEG_SIM,
+    MAX_NEG_SIM,
+    MAX_POS_SIM,
+    EMBED_DIM,
+    DROPRATE_DIALOGUE,
+    DROPRATE_LABEL,
+    DROPRATE_ATTENTION,
+    KEY_RELATIVE_ATTENTION,
+    VALUE_RELATIVE_ATTENTION,
+    MAX_RELATIVE_POSITION,
 )
```

Comment:
Just a small optional suggestion: since you're using 60% of all the constants defined in `constants.py`, maybe it's worth importing it like `import rasa.utils.tensorflow.constants as constants`?

Reply:
I personally prefer the current way